src/api.cc

```diff
@@ -164,12 +164,16 @@ AkliteClient::~AkliteClient() {
 
 static bool compareTargets(const TufTarget& a, const TufTarget& b) { return a.Version() < b.Version(); }
 
-// Returns a sorted list of targets matching tags if configured and hwid (or one of secondary_hwids)
+// Returns a sorted list of OSTREE targets matching tags if configured and hwid (or one of secondary_hwids)
 static std::vector<TufTarget> filterTargets(const std::vector<TufTarget>& allTargets, const std::string& hwidToFind,
                                             const std::vector<std::string>& tags,
                                             const std::vector<std::string>& secondary_hwids) {
   std::vector<TufTarget> targets;
   for (const auto& t : allTargets) {
+    if (t.Custom()["targetFormat"] != "OSTREE") {
```

Comment:
It's almost impossible for a non-OSTree target to slip through the overall target flow up to this point.
If it occurs then it means that something is really off, so I would print an error message here and printing target name along with the target format. Yet, still continue as you've done here.

Reply:
Done.